x/ibc/03-connection/types/connection.go

```diff
@@ -98,23 +98,13 @@ func (c Counterparty) GetPrefix() commitmentexported.Prefix {
 // ValidateBasic performs a basic validation check of the identifiers and prefix
 func (c Counterparty) ValidateBasic() error {
 	if err := host.ConnectionIdentifierValidator(c.ConnectionID); err != nil {
-		return sdkerrors.Wrap(err,
-			sdkerrors.Wrapf(
-				ErrInvalidCounterparty,
-				"invalid counterparty connection ID %s", c.ConnectionID,
-			).Error(),
-		)
+		return sdkerrors.Wrap(err, "invalid counterparty connection ID")
 	}
 	if err := host.ClientIdentifierValidator(c.ClientID); err != nil {
-		return sdkerrors.Wrap(err,
-			sdkerrors.Wrapf(
-				ErrInvalidCounterparty,
-				"invalid counterparty client ID %s", c.ClientID,
-			).Error(),
-		)
+		return sdkerrors.Wrap(err, "invalid counterparty client ID")
```

Comment:
Having the invalid identifier printed in the error would be nice

Reply:
it's already in the error 👍